src/_rules/border.js

```diff
@@ -18,6 +18,7 @@ const borderStyles = [
 export const borders = [
   [/^border$/, handlerBorder],
   [/^border-transparent$/, () => ({ 'border-color': 'transparent' })],
+  [/^border-inverted$/, () => ({ 'border-color': 'var(--w-s-border-inverted)' })],
```

Comment:
Isnt the class names the only thing running the s- prefix? (not the vars? )

Reply:
Hmm I see a lot og `--w-s-xxx` vars here: https://assets.finn.no/pkg/@warp-ds/tokens/1.0.0-alpha.92/finn-no.css - so I think it should contain -s here?